modules/models/src/main/java/org/eclipse/xpanse/modules/models/view/ProviderOclVo.java

```diff
@@ -29,7 +29,7 @@ public class ProviderOclVo {
     private List<Region> regions;
 
     @NotNull
-    @Schema(description = "The list of the registered services.")
-    private List<OclDetailVo> details;
+    @Schema(description = "The list of the deployable services.")
```

Comment:
```suggestion
    @Schema(description = "The list of the available/managed services.")
```
I suggest using available or managed services here.